### PR TITLE
Feat: Avatar alert badges

### PR DESCRIPTION
**Pull request by Rohanmond**

### CODE REVIEW

**Review by Aman-Jain1996 — APPROVED** [review]
Looks good to be merged , pls check landing page button it's not working (if not implemented yet then good to go)

**Review by vinc3nati — APPROVED** [review]
Looks good!

File: components/avatar/avater.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        <img class="avatar-circle" src="../../src/avatar/avatar3.jpeg" alt="">
+    </div>
+    <div class="avatar-container-lg">
+        <img class="avatar-square" src="../../src/avatar/avatar1.jpeg" alt="">
+    </div>
+    <div class="avatar-container-md">
+        <img class="avatar-square" src="../../src/avatar/avatar2.jpeg" alt="">
+    </div>
+    <div class="avatar-container-sm">
+        <img class="avatar-square" src="../../src/avatar/avatar3.jpeg" alt="">
```
Comment (anuj-kumary) [inline components/avatar/avater.html:27]: Always give alternate in both (avatar,badge).

Comment (Rohanmond) [reply components/avatar/avater.html:27]: Thanks it is fixed now

**Pull request merged**